src/main/java/io/wispforest/owo/Owo.java

```diff
@@ -83,4 +81,4 @@ public static MinecraftServer currentServer() {
         return SERVER;
     }
 
-}
\ No newline at end of file
+}
```

Comment:
small nitpick, does this repository use this style?

Reply:
I neither know nor care